app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt

```diff
@@ -234,4 +235,5 @@ enum class KeyboardLayout(val keyboardDefinition: KeyboardDefinition) {
     ENHRMessagEase(KB_EN_HR_MESSAGEASE),
     CZprogrammerMessagease(KB_CZ_PROG),
     ENDEThumbKeyV2(KB_EN_DE_THUMBKEY_V2),
+    ENESCA_THUMBKEY(KB_EN_ES_CA_THUMBKEY),
```

Comment:
Make sure this one is named like the others, no underscore.

Reply:
I think I've matched the naming convention with my latest commit.